crates/wasi-nn/examples/classification-example/src/main.rs

```diff
@@ -54,19 +75,45 @@ pub fn main() {
     }
     println!(
         "Found results, sorted top 5: {:?}",
-        &sort_results(&output_buffer)[..5]
+        &sort_results(&output_buffer, backend)[..5]
     )
 }
 
+fn create_gba (backend: u8) ->Vec<Vec<u8>>  {
+    let result: Vec<Vec<u8>> = match backend {
+        wasi_nn::GRAPH_ENCODING_OPENVINO => {
+            let xml = fs::read_to_string("fixture/model.xml").unwrap();
+            let weights = fs::read("fixture/model.bin").unwrap();
+            Vec::from([xml.into_bytes(), weights])
+        },
+        wasi_nn::GRAPH_ENCODING_TENSORFLOW => {
+            let model_path: String = env!("MAPDIR").to_string();
+            Vec::from([model_path.into_bytes(),
+                        "signature,serving_default".to_owned().into_bytes(),
+                        "tag,serve".to_owned().into_bytes(),
+                        ])
```

Comment:
I think this custom comma-delimited parsing should be removed like we talked about.